Procfile

```diff
@@ -0,0 +1,2 @@
+web: env/bin/python InvenTree/manage.py runserver 0.0.0.0:$PORT
```

Comment:
Should we be pushing a "proper" server (e.g. gunicorn) by default? My understanding is that the django server is not particularly performant?

Reply:
@SchrodingersGat oh this is still the test setup; give me a few minutes

Reply:
@SchrodingersGat now using gunicorn